db/migrate/20160906224205_change_markup_from_sensor_name_to_sensor_id.rb

```diff
@@ -0,0 +1,36 @@
+class ChangeMarkupFromSensorNameToSensorId < ActiveRecord::Migration
+  def up
+    sensor_name_typos = {
```

Comment:
@drjakob this database migration fixes several typos along with the new feature "markup via sensor id"